homeassistant/components/verisure/lock.py

```diff
@@ -45,8 +47,10 @@ def __init__(
         super().__init__(coordinator)
         self.serial_number = serial_number
         self._state = None
-        self._digits = coordinator.config.get(CONF_CODE_DIGITS)
-        self._default_lock_code = coordinator.config.get(CONF_DEFAULT_LOCK_CODE)
+        self._digits = coordinator.entry.options.get(
+            CONF_LOCK_CODE_DIGITS, DEFAULT_LOCK_CODE_DIGITS
+        )
+        self._default_lock_code = coordinator.entry.options.get(CONF_LOCK_DEFAULT_CODE)
```

Comment:
Probably better to just fetch this from the entry when used, so they are never out of sync with the entry options.